lettuce/boundary.py

```diff
@@ -85,6 +99,7 @@ def __init__(self, lattice, direction):
             self.w = self.lattice.w[self.velocities]
 
     def __call__(self, f):
+        # overwrite streamed fs with stored fs
```

Comment:
Why call them stored fs? We calculate them as a response to the current flow situation on the boundary, don't we?

Reply:
ah, that's left over from intermediate code
will be deleted